infra/general/iniUtils.php

```diff
@@ -7,21 +7,35 @@ class iniUtils
 {
 	/**
 	 * Given an associative array, this function will generate INI file string that represent it.
-	 * @param array $iniData
+	 * @param $ini
+	 * @param bool $isBaseLevel
+	 * @param null $baseKey
 	 * @return string
 	 */
-	public static function arrayToIniString(array $iniData)
+	public static function arrayToIniString($ini, $isBaseLevel = true, $baseKey = null)
```

Comment:
Why did you remove the type of $ini? this function works only on ini input, if it gets anything else then something is wrong